src/animations/FadeInOut/FadeInOut.scss

```diff
@@ -0,0 +1,27 @@
+@keyframes fadeIn {
+  0% {
+    opacity: 0;
+  }
+
+  100% {
+    opacity: 100%;
```

Comment:
Opacity takes values from `0` to `1` therefore, this should be `1`.

Reply:
I'm not seeing the fade effect on either chrome or firefox. Just getting an almost show/hide effect i.e white screen then content